docs/responsive-styles.md

```diff
@@ -59,7 +59,7 @@ Using Styled System with a CSS-in-JS library will generate something like the fo
 
 ## Using objects
 
-Alternatively, you can define breakpoints with aliases and use plain objects as values.
+Alternatively, you can define breakpoints with aliases and use plain objects as values. Use the alias `_` to define the base value.
```

Comment:
Thanks for this. In light of this API changing in the next version, and since *any* undefined key works in the current version, could you change the `_` key to `0`

Reply:
@jxnblk done! :)